tools/debian_buildenv.sh

```diff
@@ -27,6 +27,13 @@ case "$COMMAND" in
         esac
 
         sudo apt-get update
+        if [ $(dpkg-query -W -f='${Status}' jackd2 2>/dev/null | grep -c "ok installed") -eq 1 ]; 
+		# If jackd2 is installed as per dpkg database, install libjack-jackd2-dev.
```

Comment:
This comment still doesn't tell me why this code is here. What would be the problem if it wasn't there?

Reply:
Comments should explain *why* rather than *how*

Reply:
The comment is better now. It's pretty unusual to put a comment in the middle of an `if...then` clause. Please move it above the `if`